api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/DiagnosticReportTranslatorImpl.java

```diff
@@ -82,11 +82,7 @@ public Obs toOpenmrsType(DiagnosticReport diagnosticReport) {
 	
 	@Override
 	public Obs toOpenmrsType(Obs existingObs, DiagnosticReport diagnosticReport) {
-		if (existingObs == null) {
-			existingObs = new Obs();
-		}
-		
-		if (diagnosticReport == null) {
```

Comment:
This is actually a behaviour change. Do you think this is what we want to do? I could see:

```java
if (diagnosticReport == null) {
	return null;
}

if (existingObs == null) {
	existingObs = new Obs();
}
```